<launch>

  <node name="server2topic" pkg="socket" type="server2topic" output="screen"/>

  <node name="soundplay_node" pkg="sound_play" type="soundplay_node.py" output="screen"/>

  <node name="recog" pkg="pi_speech_tutorial" type="person_recog.py" output="screen">
  	<param name="voice" value="voice_cmu_us_clb_arctic_clunits"/>
  	<param name="wavepath" value="$(find pi_speech_tutorial)/sounds"/>
	
  </node>

</launch>
